lib/solarwinds_apm/otel_config.rb

```diff
@@ -102,16 +108,11 @@ def self.initialize
 
       print_config if SolarWindsAPM.logger.level.zero?
 
+      # resolve OTEL environmental variables
       ENV['OTEL_TRACES_EXPORTER'] = 'none' if ENV['OTEL_TRACES_EXPORTER'].to_s.empty?
+      ENV['OTEL_LOG_LEVEL'] = LEVEL_LOGGER_MAPPING[SolarWindsAPM.logger.level] if ENV['OTEL_LOG_LEVEL'].to_s.empty?
```

Comment:
This is doing my head in ;p

From the mapping https://github.com/solarwinds/apm-ruby/blob/441d544216f9d757e28155ca578fb1747d30743f/lib/solarwinds_apm/otel_config.rb#L18-L23 it seems the supported `OTEL_LOG_LEVEL` values are `debug,info,warn,error,fatal`...is this defined anywhere in the OTel specs or taken from upstream [ForwardingLogger](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/sdk/lib/opentelemetry/sdk/forwarding_logger.rb) (which also supports the value `unknown`), or detemined some other way?

So IIRC firs we set the SolarWindsAPM.logger.level based on this mapping of `SW_APM_DEBUG_LEVEL`-to-Ruby logger levels https://github.com/solarwinds/apm-ruby/blob/bd23132e6448c2911bd6cb79b4e65b1a560211ca/lib/solarwinds_apm/config.rb#L15-L22

Then in line 113 we are setting `OTEL_LOG_LEVEL` by mapping the SolarWindsAPM.logger.level, now the integers 0-5 (debug to unknown per https://github.com/ruby/ruby/blob/master/lib/logger/severity.rb), to an accepted string value... and `unknown` is skipped because we currently don't map any SW debug level to Ruby's Logger::UNKNOWN level?

Is it possible to use a single mapping structure so that this convolution is a little clearer, something like below, which can be used when setting SolarWindsAPM.logger.level and OTEL_LOG_LEVEL?
```
SW_LOG_LEVEL_MAPPING = {-1 => {:stdlib => ::Logger::FATAL, :otel=>'fatal',
                         0 => {:stdlib => ::Logger::FATAL, :otel=>'fatal',
                         1 => {:stdlib => ::Logger::ERROR, :otel=>'error',                       
                        ...}.freeze 
```

Reply:
Yes, I was struggling on mapping log level to achieve log level sync. I have changed to the `SW_LOG_LEVEL_MAPPING`